changelog.md

```diff
@@ -6,8 +6,12 @@
 
 ## Bug fixes
 
-* Fix reporting error based on http responses from the Aura-API with an invalid JSON body. Earlier the client would report JSONDecodeError instead of showing the actual issue.
+- Fix reporting error based on http responses from the Aura-API with an invalid JSON body. Earlier the client would report JSONDecodeError instead of showing the actual issue.
 
 ## Improvements
 
+- `GraphDataScience::run_query` now supports setting the `mode` of the query to be used for routing. Previously queries would always route the leader of the cluster, assuming write mode.
```

Comment:
- The following endpoints will now run in Cypher `READ` mode instead of `WRITE` mode:
    - `gds.graph.cypher.project()`
    - `gds.graph.nodeProperties.stream()`